Sources/MuxUploadSDK/Internal Utilities/UploadEvent.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  UploadEvent.swift
+//  
+//
+//  Created by Liam Lindner on 3/22/23.
+//
+
+import Foundation
+import UIKit
+
+struct UploadEvent: Codable {
+    var type = "upload"
+    var start_time: TimeInterval
```

Comment:
These can be camel cased while still encoding to JSON with snake cased keys if the `JSONEncoder` has its `keyEncodingStrategy` set to `convertToSnakeCase`.

Reply:
Thanks! Fixed